resilience4j-core/src/main/java/io/github/resilience4j/core/IntervalBiFunction.java

```diff
@@ -0,0 +1,18 @@
+package io.github.resilience4j.core;
+
+import io.vavr.control.Either;
```

Comment:
We are trying to remove `vavr` dependency in 2.0 branch.
Is there any substitute for`Either`?

Reply:
We have to provide our own implementation in 2.0

Reply:
just for my personal curiosity, why do you want to get rid of `vavr`?

Reply:
Because a lot of users requested it, unfortunately.